### ISSUE TITLE
Surround object references in docstrings with backticks to allow Sphinx to generate links.

### ISSUE THREAD

**Issue by KelSolaar** [vispy/vispy#912]
Object references in the docstrings should be surrounded with backticks in order to allow Sphinx to generate links:

For example: https://github.com/vispy/vispy/blob/3e21db8f91619c103c29b0f4e9032383cd3a2015/vispy/scene/visuals.py#L7

```
These classes are automatically generated by mixing vispy.scene.Node with
the Visual classes found in vispy.visuals.
```

Could be written as follows:

```
These classes are automatically generated by mixing `vispy.scene.Node` with
the Visual classes found in `vispy.visuals`.
```

References
---
- https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#class-docstring

**Comment by Eric89GXL** [comment]
TL;DR: single-backticks for things that should have references, double-backticks for code literals

**Comment by Eric89GXL** [comment]
I took care of a bunch of these in recent PRs, I'm going to go ahead and close this for now. We should also keep in mind `See also` sections, and also `..versionadded` directives.
